matchbot-cli.php

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env php
```

Comment:
see https://github.com/thebiggive/matchbot/pull/639/files#r1367150515 . Changed file mode to executable.